{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/constants/metadata.py

```diff
@@ -19,6 +19,16 @@
     # TODO - project locations here
 ]
 
+ARTIFACT_INDEX_COLUMNS = [
+    'sex',
+    'age_start',
+    'age_end',
+    'year_start',
+    'year_end',
+]
+
+ARTIFACT_COLUMNS = pd.Index([f'draw_{i}' for i in range(0, 1000)])
+
```

Comment:
is the `YEAR_DURATION` constant too project-specific, so we don't want to add it to the base template?